docs/user/custom_domains.rst

```diff
@@ -85,6 +85,14 @@ for our own domains including ``*.readthedocs.io``, ``*.readthedocs-hosted.com``
 
 .. _Strict Transport Security header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
 
+Multiple documentation sites as sub-folders of a domain
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+You may host multiple documentation repositories as **sub-folders of a single domain**.
+For example, ``docs.foo.org/repo1`` and ``docs.foo.org/repo2``.
```

Comment:
```suggestion
For example, ``docs.example.org/projects/repo1`` and ``docs.example.org/projects/repo2``.
```

Unfortunately this is still a limitation of this feature, so we don't end up with weird URL overlap. We can and should probably remove this at some point.

Reply:
ah so it **must** have `projects/` in it?

Reply:
Yes. This is a current limitation 😢

Reply:
ah bummer, that is indeed a big limitation